Task_DEV-9/src/SymbolsSwap.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Random;
+
+/**
+ * Swaps symbols in strings.
+ */
+public class SymbolsSwap {
+
+  /**
+   * Gets a piece of the second string and puts instead another piece in the first string.
+   * Length of string pieces, places of cutting and insertion are chosen randomly.
+   * @param firstString a string we gets to insert symbols into it.
+   * @param secondString a string we gets to cut symbols from it.
+   * @return first string after swapping.
+   */
+  public String swapPiecesRandomly(String firstString, String secondString) throws java.lang.IllegalArgumentException {
```

Comment:
Hard to read due to the extremely long names and lines, also method can be split to a few methods